### DISCUSSION TITLE
Colors on FMS

### DISCUSSION THREAD

**Discussion by rpoulachon** [flybywiresim/aircraft#4200] [General]
Hi,

I am wondering if colors for text in FMS display are similar to the true FMS in real plane ? As I can see in different videos,  red color for example is more intense or even blue text color seems to be different.
Perharps I have missed something.

Thanks and sorry for my english.

**Comment by tracernz** [comment]
There is a difference between Honeywell and Thales units I believe. I can’t comment on realism as I’ve only seen videos, never the real thing, but we do have a large number of real pilots on our Discord checking and giving feedback all the time.

**Reply by rpoulachon** [reply]
Thank you I will exchange with them on discord.
